scripts/aws/cloudformation/tasking-manager.template.js

```diff
@@ -359,80 +362,155 @@ const Resources = {
+          'sudo apt-get -q -y install python3.9 python3-pip python3-psycopg2 python-is-python3',
+          'sudo apt-get -q -y install postgresql-client postgis libpq-dev',
+          'sudo apt-get -q -y install libxml2 libxml2-dev',
+          'sudo apt-get -q -y install libgeos-3.9.0 libgeos-dev',
+          'sudo apt-get -q -y install libproj19 libproj-dev',
+          'sudo apt-get -q -y install libjson-c-dev',
+          'git clone --recursive https://github.com/hotosm/tasking-manager.git /opt/tasking-manager',
+          'cd /opt/tasking-manager/',
+          cf.sub('git reset --hard ${GitSha}'),
+          'pip install --upgrade pip pdm==2.7.4',
+          'wget -6 https://s3.dualstack.us-east-1.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz -O /tmp/aws-cfn-bootstrap-py3-latest.tar.gz',
+          'pip install /tmp/aws-cfn-bootstrap-py3-latest.tar.gz',
+          'pdm install',
+          'eval "$(pdm venv activate)"',
+          'echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf',
+          'export LC_ALL=C',
+          'wget -6 https://s3.dualstack.us-east-1.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb -O /tmp/amazon-cloudwatch-agent.deb',
+          'dpkg -i /tmp/amazon-cloudwatch-agent.deb',
+          'echo "Exporting environment variables:"',
+          cf.sub('export NEW_RELIC_LICENSE=${NewRelicLicense}'),
+          cf.join('', ['export POSTGRES_ENDPOINT=', cf.getAtt('TaskingManagerRDS','Endpoint.Address')]),
+          cf.sub('export POSTGRES_DB=${PostgresDB}'),
+          cf.sub('export POSTGRES_PASSWORD="${PostgresPassword}"'),
+          cf.sub('export POSTGRES_USER="${PostgresUser}"'),
+          cf.sub('export TM_APP_BASE_URL="${TaskingManagerAppBaseUrl}"'),
+          cf.sub('export TM_ENVIRONMENT="${AWS::StackName}"'),
+          cf.sub('export TM_CLIENT_ID="${TaskingManagerOAuthClientID}"'),
+          cf.sub('export TM_CLIENT_SECRET="${TaskingManagerOAuthClientSecret}"'),
+          cf.sub('export TM_REDIRECT_URI="${TaskingManagerAppBaseUrl}/authorized"'),
+          'export TM_SCOPE="read_prefs write_api"',
+          cf.sub('export TM_SECRET="${TaskingManagerSecret}"'),
+          cf.sub('export TM_SMTP_HOST="${TaskingManagerSMTPHost}"'),
+          cf.sub('export TM_SMTP_PASSWORD="${TaskingManagerSMTPPassword}"'),
+          cf.sub('export TM_SMTP_PORT="${TaskingManagerSMTPPort}"'),
+          cf.sub('export TM_SMTP_USER="${TaskingManagerSMTPUser}"'),
+          cf.sub('export TM_SMTP_USE_SSL="${TaskingManagerSMTPSSL}"'),
+          cf.sub('export TM_SMTP_USE_TLS="${TaskingManagerSMTPTLS}"'),
+          cf.sub('export TM_DEFAULT_CHANGESET_COMMENT="${TaskingManagerDefaultChangesetComment}"'),
+          cf.sub('export TM_EMAIL_FROM_ADDRESS="${TaskingManagerEmailFromAddress}"'),
+          cf.sub('export TM_EMAIL_CONTACT_ADDRESS="${TaskingManagerEmailContactAddress}"'),
```

Comment:
value in CFN set to an email address we don't monitor. So maybe we should let this fallback to defaults. Good to remove.